content/raw/chat/username_avatar.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "username & profile spec"
+description: ""
+date: 2020-12-06T08:48:57+00:00
+lastmod: 2020-12-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Username & Profile Spec
+
+## Motivation
+
+Users need a mechanism to identity themselves to other users that is easy to use and at the same time somewhat resistant to impersonation attacks. At the time of writing Status uses deterministic 3 word usernames and ENS usernames. Unfortunatly 3 word usernames have shown to be quite confusing for new users as they are used to set their own usernames in any chat app.
```

Comment:
More background context on the problems these design changes are aiming to solve is documented here https://docs.google.com/document/d/1d5zdPR2_Jn2EZmKNQVNsc7sUoLk4DXPN7Gch3CnYkaE/edit